generators/client/templates/README.md.jhi.client.ejs

```diff
@@ -28,21 +28,20 @@
 _&>
 <%_
 let clientPackageMngrName = 'Npm';
+let clientPackageMngrWrapper = './npmw';
 let clientPackageMngrAddGlobal = 'install -g';
 let clientPackageMngrAdd = 'install --save --save-exact';
 let clientPackageMngrAddDev = 'install --save-dev --save-exact';
 -%>
-<&_ if (fragment.developmentSection) { -&>
-Before you can build this project, you must install and configure the following dependencies on your machine:
 
-1. [Node.js](https://nodejs.org/): We use Node to run a development web server and build the project.
-   Depending on your system, you can install Node either from source or as a pre-packaged bundle.
+The build system will install automatically the right version of node and <%= clientPackageManager %>.
```

Comment:
Updated documentation should be added if `useNpmWrapper` is true otherwise old documentation should be generated.

Reply:
Fixed